src/index.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
 import * as serviceWorkerRegistration from './serviceWorkerRegistration';
 import reportWebVitals from './reportWebVitals';
+import BotNav from './Componants/navbar';
```

Comment:
Can we fix the spelling on the directory? Will need to rename the actual directory too.
```suggestion
import BotNav from './Components/navbar';
```

Reply:
Did not even take notice. Was trying to do it too fast. My bad.